### ISSUE TITLE
IntelliJ PyCharm and Kite Copilot

### ISSUE THREAD

**Issue by lovegandhi** [kiteco/plugins#115]
**System**: I am running Kite 20181109, Plugin v1.1.194 in PyCharm 2018.2
**Issue**: Kite Copilot does not show anything related to my project. By "does not show anything" I mean that Copilot does not refresh with new data for current Class or Method under cursor/typed. Imported packages work just fine.

Simple test files will reproduce this error.
file1.py:
```
class ClassName(object):
    """
    Documentation for ClassName, this is just a test file
    """
    def testing_method(self, parameter):
        """
        Documentation for testing method. This is just a test file
        :param parameter:
        :return:
        """
        pass
```
file2.py:
```
import file1

class_name = file1.ClassName()
class_name.testing_method()
```
**Expectation**: I expect, and used to work before, that when I type `ClassName` Copilot would display documentation and possibly how others used this method or how to.
The other expectation is that when I type `class_name = testing_method|` Copilot would yet again display documentation for this method. However, it does not.

**Comment by lovegandhi** [comment]
Trying to search for this Class in Copilot does not yield any results as well. I am trying file1.ClassName or just ClassName

**Comment by dhung09** [comment]
Hi @lovegandhi thanks for reporting this issue - we're looking into it.

To help us debug, what OS are you using?

**Comment by lovegandhi** [comment]
Hi @dhung09, I am running macOS High Sierra.

**Comment by lovegandhi** [comment]
Hi @dhung09 Any updates?

**Comment by dhung09** [comment]
Sorry for the late response. Do you mind sending your Kite logs to feedback@kite.com and we can debug further? Kite logs are located at `~/.kite/logs/client.log`. Also, does the file `~/.kite/KITE_LOCAL` exist on your machine?

Regarding searching in the Copilot: Search unfortunately does not work with your locally defined code. Apologies for the inconvenience!

**Comment by lovegandhi** [comment]
Logs sent.
~/.kite/KITE_LOCAL does not exist on my machine.

**Comment by lovegandhi** [comment]
Issues seems to have been fixed in Kite v0.20181129.0
